src/environmentd/src/bin/environmentd/main.rs

```diff
@@ -701,7 +701,13 @@ max log level: {max_log_level}",
                     )
                 })
                 .filter(|(name, _value)| name.starts_with("MZ_") || name == "FAILPOINTS")
-                .map(|(name, value)| format!("{}={}", escape(&name), escape(&value)))
+                .map(|(name, value)| {
+                    let value = match name.as_str() {
+                        "MZ_LAUNCHDARKLY_SDK_KEY" => "<masked>".to_string(),
+                        _ => value,
+                    };
+                    format!("{}={}", escape(&name), escape(&value))
+                })
```

Comment:
Would you actually mind to just delete this entire `println!`? We're leaking secrets left and right here (the Cockroach passwords come in as env vars, for example). This made sense in the binary days, but these days, it's just as easy to pull this information from Kubernetes (if not easier).